### ISSUE TITLE
Mounting NFS Shared folders on windows 10

### ISSUE THREAD

**Issue by carlead** [winnfsd/vagrant-winnfsd#117]
Hi,

I have an ubuntu vagrant box on windows 10 guest, I installed plugin
vagrant plugin install vagrant-winnfsd
and on ubuntu I did
sudo apt-get install nfs-kernel-server

But with vagrant up I'm stuck at 
==> default: Mounting NFS shared folders...

`config.vm.network "private_network", type: "dhcp"
config.vm.synced_folder ".", "/var/www", type: "nfs"
`

Thanks,

**Comment by Yamaha32088** [comment]
I had the same issue and it ended up being the windows firewall. You can **TEMPORARILY** disable the windows firewall for all networks and see if that fixes it. If it does make sure to re-enable the firewall and allow the winnfsd service through the firewall.

**Comment by ohick** [comment]
@Yamaha32088 Thanks it worked for me. After updating winnfsd I had 2 versions. I gave the firewall permission to all of them and it worked.

**Comment by func0der** [comment]
Yeah, this plugin ususally adds firewall rules by itself. That seems broken.
We need a fix for this.

**Comment by performize** [comment]
I had Eset NOD32 Internet security  installed and I had to allow nfs and vmware inside the firewall settings. Antivirus in automatic mode doesn't alert you and it block all connections by default.
Connection timeout disappear immediately after allowing connections.
